spec/services/organization_update_service_spec.rb

```diff
@@ -51,12 +53,12 @@
           partner_two.profile.reload
 
           aggregate_failures "request type in organization and partners" do
-            expect(organization.enable_individual_requests).to eq(true)
-            expect(partner_one.profile.enable_individual_requests).to eq(true)
-            expect(partner_two.profile.enable_individual_requests).to eq(true)
-            expect(organization.enable_child_based_requests).to eq(true)
-            expect(partner_one.profile.enable_child_based_requests).to eq(true)
-            expect(partner_two.profile.enable_child_based_requests).to eq(true)
+            expect(organization.enable_individual_requests).to eq(organization.errors.none?)
```

Comment:
You should know in the test whether there were errors or not. If you want to test this in an error vs not error state, they should be two separate explicit tests.